src/pages/ProductItem/ProductItem.scss

```
@@ -0,0 +1,175 @@
+.layout {
+  // background-color: yellow;/
+  display: flex;
+  flex-direction: column;
+
+  main {
+    // background-color: red;
+    display: flex;
+    padding-bottom: 70px;
+    margin: 0px 110px;
+    margin-top: 50px;
```

아래와 같이 작성하실 수 있습니다. 
- 더불어 가장 상위 레이아웃의 여백을 고정 margin px단위로 주시면 해당 화면을 보는 디스플레이의 크기에 따라 모두 다른 형태로 보여질 수 있을 것 같습니다. 제 디스플레이에서는 왼쪽으로 치우쳐저 있네요. 왜 그럴지, css어떤 속성을 이용하면 좋을지 생각해보시고 댓글 달아주세요!
```suggestion
    margin: 50px 110px 0px;
```

---

아...
margin으로 여백을 줄 것이 아니라
flex-box를 사용해서 center를 잡는 방법으로 가야했던 것 같습니다!
각 디스플레이들 마다 넓이가 다르므로, 
가장 상위 레이아웃에서 margin으로 여백을 주는 것은 
모두 다른 넓이를 가진 디스플레이에서, 각각 여백을 달라지게하겠내요 ㅠㅠ